src/views/ThreeWActivityForm/schema.ts

```diff
@@ -134,6 +134,7 @@ const finalSchema: FormSchema = {
             ['activities', 'sectors'] as const,
             ['sectors'] as const,
             (val): Pick<FormSchemaFields, 'sectors'> => {
+                // FIXME: This condition is not triggered after value change
```

Comment:
This has been fixed. Let's remove the FIXME.
The issue was caused by improper usage of setValue function